zrml/swaps/src/lib.rs

```diff
@@ -105,7 +108,12 @@ mod pallet {
     >>::Balance;
     type MarketIdOf<T> = <<T as Config>::MarketCommons as MarketCommonsPalletApi>::MarketId;
 
+    pub(crate) const ARBITRAGE_MAX_ITERATIONS: usize = 30;
+    const ARBITRAGE_MIN_WEIGHT: Weight = ON_IDLE_MIN_WEIGHT / (ARBITRAGE_WEIGHT_RATIO as u64);
+    const ARBITRAGE_THRESHOLD: u128 = CENT;
+    const ARBITRAGE_WEIGHT_RATIO: u32 = 2;
     const MIN_BALANCE: u128 = CENT;
+    const ON_IDLE_MIN_WEIGHT: Weight = 1_000_000;
```

Comment:
"I made them up," except for `ARBITRAGE_MAX_ITERATIONS`, which is explained here: https://hackmd.io/WXQGKQglS8O2cKqSKwMMxA#How-many-iterations

To be honest, I feel like getting rid of `ARBITRAGE_MIN_WEIGHT`. I don't know why I even started with that.

The threshold of `CENT` makes sense to me, it basically means if you truncate after the second digit after the decimal point, the sum of all spot prices is 1.00.

Reply:
So the point about the maximum number of iterations is that we have a mathematical guarantee that unless balances go above one million, arbitrage is done after 30 iterations. So the idea is to make sure that in case of an error in implementation, we don't do any more iterations that we have to.

Reply:
`ARBITRAGE_MIN_WEIGHT` removed: 93d23a577cb0c100dd49ce0728958e7cca942f95

Reply:
Ok, brilliant (`ARBITRAGE_MAX_ITERATIONS`), so in case that balances go above one million (which is surely rare), the arbitrage could happen more than 30 times? Ok, so `ARBITRAGE_THRESHOLD` defines a type of precision (good point, I think it precise enough).

Reply:
No, arbitrage is always limited to 30 iterations. But if a balance goes over 1 million, then we don't reach the precision of 1 mZTG for the arbitrage amount. The precision kind of scales back linearly: If you have a balance of x million, the precision is now x mZTG.

Reply:
Thanks for explaining that. I am glad, that we have the hard limit of iterations in the blockchain.